Analysis/include/Luau/ControlFlow.h

```diff
@@ -14,9 +14,24 @@ enum class ControlFlow
     None = 0b00001,
     Returns = 0b00010,
     Throws = 0b00100,
-    Break = 0b01000,    // Currently unused.
-    Continue = 0b10000, // Currently unused.
+    Breaks = 0b01000,
+    Continues = 0b10000,
+
+    // Returns OR Throws (e.g. if predicate then return else error() end)
+    MixedFunctionExit = 0b100000,
+
+    // Breaks OR Continues (e.g. if predicate then break else continue end)
+    MixedLoopExit = 0b1000000,
+
+    // Exits a loop OR function (e.g. if prediate then continue else return end)
+    MixedExit = 0b10000000,
```

Comment:
The reason for why we don't return a mixed control flow signal was because it would be able to supersede `Luau::getFallthrough` that I never got a chance to do yet.

It's my personal recommendation to clip the rest.
```suggestion
```